.vscode/cspell.json

```
@@ -18,8 +22,134 @@
+    ".vscode/cspell.json",
+    "sdk/**/arm-*/**",
+    "sdk/test-utils/**",
+    "sdk/agrifood/agrifood-farming-rest/review/*.md",
+    "sdk/apimanagement/arm-apimanagement/review/*.md",
+    "sdk/appservice/arm-appservice/review/*.md",
+    "sdk/authorization/arm-authorization/review/*.md",
+    "sdk/compute/arm-compute/review/*.md",
+    "sdk/confidentialledger/confidential-ledger-rest/review/*.md",
+    "sdk/core/core-client-lro-rest/review/*.md",
+    "sdk/core/core-client-paging-rest/review/*.md",
+    "sdk/core/core-client-rest/review/*.md",
+    "sdk/documenttranslator/ai-document-translator-rest/review/*.md",
+    "sdk/eventgrid/arm-eventgrid/review/*.md",
+    "sdk/eventhub/arm-eventhub/review/*.md",
+    "sdk/extendedlocation/arm-extendedlocation/review/*.md",
+    "sdk/features/arm-features/review/*.md",
+    "sdk/keyvault/arm-keyvault/review/*.md",
+    "sdk/links/arm-links/review/*.md",
+    "sdk/locks/arm-locks/review/*.md",
+    "sdk/managedapplications/arm-managedapplications/review/*.md",
+    "sdk/network/arm-network/review/*.md",
+    "sdk/operationalinsights/arm-operationalinsights/review/*.md",
+    "sdk/policy/arm-policy/review/*.md",
+    "sdk/purview/arm-purview/review/*.md",
+    "sdk/purview/purview-account-rest/review/*.md",
+    "sdk/purview/purview-administration-rest/review/*.md",
+    "sdk/purview/purview-catalog-rest/review/*.md",
+    "sdk/purview/purview-scanning-rest/review/*.md",
+    "sdk/quantum/quantum-jobs/review/*.md",
+    "sdk/quota/arm-quota/review/*.md",
+    "sdk/redis/arm-rediscache/review/*.md",
+    "sdk/resources/arm-resources/review/*.md",
+    "sdk/resources-subscriptions/arm-resources-subscriptions/review/*.md",
+    "sdk/security/arm-security/review/*.md",
+    "sdk/servicebus/arm-servicebus/review/*.md",
+    "sdk/sql/arm-sql/review/*.md",
+    "sdk/storage/arm-storage/review/*.md",
+    "sdk/synapse/synapse-access-control/review/*.md",
+    "sdk/synapse/synapse-artifacts/review/*.md",
+    "sdk/synapse/synapse-managed-private-endpoints/review/*.md",
+    "sdk/synapse/synapse-monitoring/review/*.md",
+    "sdk/synapse/synapse-spark/review/*.md",
+    "sdk/templatespecs/arm-templatespecs/review/*.md",
+    "sdk/web-pubsub/arm-webpubsub/review/*.md",
+    "sdk/attestation/attestation/review/**/*.md",
+    "sdk/communication/communication-common/review/**/*.md",
+    "sdk/containerregistry/container-registry/review/**/*.md",
+    "sdk/core/core-amqp/review/**/*.md",
+    "sdk/cosmosdb/cosmos/review/**/*.md",
+    "sdk/digitaltwins/digital-twins-core/review/**/*.md",
+    "sdk/formrecognizer/ai-form-recognizer/review/**/*.md",
+    "sdk/identity/identity/review/**/*.md",
+    "sdk/iot/iot-modelsrepository/review/**/*.md",
+    "sdk/keyvault/keyvault-certificates/review/**/*.md",
+    "sdk/keyvault/keyvault-keys/review/**/*.md",
+    "sdk/monitor/monitor-query/review/**/*.md",
+    "sdk/search/search-documents/review/**/*.md",
+    "sdk/storage/storage-blob/review/**/*.md",
```

This list seems complicated to maintain. Would `**/review/**` be possible? What about having something (like a script) generate the list for us?

---

How about `sdk/*/*/review/**/*.md`?

---

I would amend that to below. Assuming this pattern gets copy/pasted, there are repos like go that don't have a consistent directory nesting level under `sdk`.

```suggestion
sdk/**/review/**/*.md
```

---

JSON doesn't allow comments. This list is made up of 2 sets: 

1. Packages which are completely auto generated and don't have any handwritten parts. Spell checking here is happening further upstream. 
2. Packages which have spelling error detections in the public API surface. These issues need to be fixed. There will soon come a collection of bugs like https://github.com/Azure/azure-sdk-for-js/issues/17143 that detail the spelling errors and what to do about them.